tasks/managed/update-cr-status/tests/test-update-cr-status-multiple-result-files.yaml

```diff
@@ -53,6 +94,37 @@ spec:
                 releasePlan: foo
               EOF
               kubectl apply -f release
+          - name: skip-trusted-artifact-operations-array
```

Comment:
maybe we can have `stepactions/skip-trusted-artifact-operations/skip-trusted-artifact-operations.yaml` and `stepactions/skip-trusted-artifact-operations/skip-trusted-artifact-operations-array.yaml`?

Reply:
I was able to convert `create-trusted-artifact-array` and `use-trusted-artifact-array`.

However, `patch-source-data-artifact-result-array` cannot be converted due to https://github.com/tektoncd/pipeline/issues/2374